<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-TEJ5K7T2" ignored="0" name="Test" tests="22" failures="9" timestamp="2021-12-20T12:14:57 IST" time="206.242" errors="0">
  <testcase name="getImageOnScreen" time="9.468" classname="Qualitest.qtrecog.ActivityPage">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Qualitest.qtrecog.ActivityPage.getImageOnScreen(ActivityPage.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getImageOnScreen -->
  <testcase name="getNameOnScreen" time="7.529" classname="Qualitest.qtrecog.ActivityPage">
    <failure type="java.lang.AssertionError" message="expected [P Akshay Raghotham] but found [Madhu Sudhan M]">
      <![CDATA[java.lang.AssertionError: expected [P Akshay Raghotham] but found [Madhu Sudhan M]
at Qualitest.qtrecog.ActivityPage.getNameOnScreen(ActivityPage.java:48)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getNameOnScreen -->
  <testcase name="titleName" time="1.476" classname="Qualitest.qtrecog.ActivityPage">
    <failure type="java.lang.AssertionError" message="expected [QTRecognition] but found []">
      <![CDATA[java.lang.AssertionError: expected [QTRecognition] but found []
at Qualitest.qtrecog.ActivityPage.titleName(ActivityPage.java:71)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- titleName -->
  <testcase name="validLogin" time="9.37" classname="Qualitest.qtrecog.ActivityPage"/>
  <testcase name="kudosFrom" time="11.399" classname="Qualitest.qtrecog.OthersTestCases"/>
  <testcase name="kudosSentToday" time="22.263" classname="Qualitest.qtrecog.OthersTestCases"/>
  <testcase name="kudosTo" time="10.877" classname="Qualitest.qtrecog.OthersTestCases"/>
  <testcase name="recentActivity" time="11.185" classname="Qualitest.qtrecog.OthersTestCases"/>
  <testcase name="selfKudos" time="16.99" classname="Qualitest.qtrecog.OthersTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Qualitest.qtrecog.OthersTestCases.selfKudos(OthersTestCases.java:115)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- selfKudos -->
  <testcase name="fetchRecent" time="15.541" classname="Qualitest.qtrecog.RecentContactsTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Qualitest.qtrecog.RecentContactsTestCases.fetchRecent(RecentContactsTestCases.java:45)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- fetchRecent -->
  <testcase name="searchKudos" time="9.339" classname="Qualitest.qtrecog.SearchKudosTest"/>
  <testcase name="searchKudos1" time="9.902" classname="Qualitest.qtrecog.SearchKudosTest"/>
  <testcase name="noCitation" time="10.777" classname="Qualitest.qtrecog.InvalidSendKudosTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Qualitest.qtrecog.InvalidSendKudosTest.noCitation(InvalidSendKudosTest.java:67)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- noCitation -->
  <testcase name="noKudosImage" time="10.268" classname="Qualitest.qtrecog.InvalidSendKudosTest"/>
  <testcase name="nonQtEmail" time="11.453" classname="Qualitest.qtrecog.InvalidSendKudosTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Qualitest.qtrecog.InvalidSendKudosTest.nonQtEmail(InvalidSendKudosTest.java:48)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- nonQtEmail -->
  <testcase name="getGold" time="3.204" classname="Qualitest.qtrecog.HomePageUrl">
    <failure type="java.lang.AssertionError" message="expected [#FECC160] but found [#FDCC16]">
      <![CDATA[java.lang.AssertionError: expected [#FECC160] but found [#FDCC16]
at Qualitest.qtrecog.HomePageUrl.getGold(HomePageUrl.java:87)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getGold -->
  <testcase name="getNavy" time="4.751" classname="Qualitest.qtrecog.HomePageUrl"/>
  <testcase name="getWhite" time="3.178" classname="Qualitest.qtrecog.HomePageUrl">
    <failure type="java.lang.AssertionError" message="expected [#FECC160] but found [#FFFFFF]">
      <![CDATA[java.lang.AssertionError: expected [#FECC160] but found [#FFFFFF]
at Qualitest.qtrecog.HomePageUrl.getWhite(HomePageUrl.java:101)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getWhite -->
  <testcase name="invalidLogin" time="4.412" classname="Qualitest.qtrecog.HomePageUrl"/>
  <testcase name="title" time="2.524" classname="Qualitest.qtrecog.HomePageUrl"/>
  <testcase name="userValidLogin" time="9.85" classname="Qualitest.qtrecog.HomePageUrl"/>
  <testcase name="userKudosLogin" time="10.434" classname="Qualitest.qtrecog.SendKudosTestCase"/>
</testsuite> <!-- Test -->
